### ISSUE TITLE
[Live] Handle the state of your bid is the current max, but you're waiting on operator to confirm

### ISSUE THREAD

**Issue by orta** [artsy/eigen#1577]
Memory serves this still looks like a max bid but with disabled states

**Comment by ashfurrow** [comment]
I can take this one – we'll need to start using the `bidPendingSignal` in `LiveAuctionBiddingViewModel`. The UI setup is already done, so it should "just work" 😄

**Comment by acjay** [comment]
Currently works as designed
